astro/src/content/quickstarts/quickstart-rust-actix-web.mdx

```diff
@@ -0,0 +1,226 @@
+- `callback` does most of the work. The user is returned to `callback` after logging in with FusionAuth. The function checks that the PKCE challenge is correct, retrieves the access token, and makes a final call to FusionAuth to get the user's email, which it stores in the session. Now the application considers the user logged in.
+
+{frontmatter.framework} automatically links the user's session to their browser by returning a cookie for the site, which is then included in every subsequent request.
+
+Now that `callback` has set a cookie, you can see how authentication on the other pages is tested. The application:
+
+- Sends the user to the account page if they already have a login cookie on the home page.
+- Sends the user to the home page if they are not logged in on the account page.
+
+## Customization
+
+With authentication done, the last task is to create example pages that a user can browse.
+
+### CSS And HTML
+
+Create a `static` directory within `your-application` directory.
+
+```shell
+mkdir static
+```
+
+Copy images from the example app.
+
+```shell
+cp ../complete-application/static/money.jpg static/money.jpg
+cp ../complete-application/static/changebank.svg static/changebank.svg
+```
+
+Create a stylesheet file `static/changebank.css` and add the following code to it.
+
+<RemoteCode
+  url={frontmatter.codeRoot + "/complete-application/static/changebank.css"}
+  lang="css"/>
+
+Next, create a `templates` directory within `your-application`. Create three pages inside the `templates` directory. First create the home page, `index.html`, and paste the following code into it.
+
+<RemoteCode
+  url={frontmatter.codeRoot + "/complete-application/templates/index.html"}
+  lang="html"/>
+
+The index page contains nothing to note except a link to the login page `<a href="/login">`.
+
+Next, create an `account.html` page and paste the following code into it.
+
+<RemoteCode
+  url={frontmatter.codeRoot + "/complete-application/templates/account.html"}
+  lang="html"/>
+
+The account page displays the user's email from FusionAuth with `<p class="header-email">{{email}}</p>`.
+
+The account page is only visible to logged in users. If a session email is not found, the user is redirected to login.
+
+Finally, create a `change.html` page and paste the following code into it.
+
+<RemoteCode
+  url={frontmatter.codeRoot + "/complete-application/templates/change.html"}
+  lang="html"/>
+
+The HTML at the bottom of the file displays a blank form when the page first loads (GET) or the result of the calculation when returning (POST).
+
+## Run The Application
+
+Run your application.
+
+```bash
+cargo run
+```
+
+Browse to the app at http://localhost:9012. Log in using `richard@example.com` and `password`. The change page allows you to enter a number. If you don't log in, you won't be able to access the change or account pages.
```

Comment:
```suggestion
Browse to the app at http://localhost:9012. Log in using <code><RemoteValue url={frontmatter.codeRoot + "/kickstart/kickstart.json"} selector="$.variables.userEmail" /></code> and <code><RemoteValue url={frontmatter.codeRoot + "/kickstart/kickstart.json"} selector="$.variables.userPassword" /></code>. The change page allows you to enter a number. If you don't log in, you won't be able to access the change or account pages.
```

Using [`<RemoteValue>`](https://github.com/FusionAuth/fusionauth-astro-components?tab=readme-ov-file#remotevalue) just in case we change these in the future :)